src/main/java/org/example/movieSpring/model/Movie.java

```diff
@@ -0,0 +1,27 @@
+package org.example.movieSpring.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "movies")
+public class Movie {
+
+    @Id
+    @Column(name = "id_movie")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "movie_title")
+    private String title;
+
+    @Column(name = "movie_release_year")
+    private int release_year;
```

Comment:
the column names dont follow convention
they should be all uppercase
none of them should include the table name
table names in columns hint at foreign keys
they can be left out and let hibernate name them in this case

i would not assign ids in the software but rather do it in the db

Reply:
"they can be left out and let hibernate name them in this case"
This I dont understand, how would I do that?

"i would not assign ids in the software but rather do it in the db"
This I think I understand, what youre saying is in MySQL when I define the column, I click in Generate checkbox on ID, and remove row 17-20 in Movie.java?
![image](https://github.com/fstetlerKnightec/MovieProject/assets/152986163/3a062155-5ec9-4df6-b680-c25bcb880315)

Reply:
> "they can be left out and let hibernate name them in this case" This I dont understand, how would I do that?
> 
> "i would not assign ids in the software but rather do it in the db" This I think I understand, what youre saying is in MySQL when I define the column, I click in Generate checkbox on ID, and remove row 17-20 in Movie.java? ![image](https://private-user-images.githubusercontent.com/152986163/331123410-3a062155-5ec9-4df6-b680-c25bcb880315.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NjQzNjAsIm5iZiI6MTcxNTg2NDA2MCwicGF0aCI6Ii8xNTI5ODYxNjMvMzMxMTIzNDEwLTNhMDYyMTU1LTVlYzktNGRmNi1iNjgwLWMyNWJjYjg4MDMxNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxNlQxMjU0MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YzM4ODYzMGI5ZDlhZWI3Y2JmNTkzNzNmMzUzZjdlNTgxYzE0YzZkNDU2YzUzODNlMDVjYzllNjAwOTczMDM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XlxnEk6zIQrNuE9KoJbNf6lHiHOyFUivMf-vGyDZwFg)

line 17 you need, jpa needs a @Id, @generatedvalue you can set strategy to auto for example

and if you do not define any column in jpa hibernate should do it for you

Reply:
done